src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +94,14 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> (p.getSex() == People.Sex.WOMEN
+                        && p.getAge() >= fromAge
```

Comment:
let's move `p.getAge() >= fromAge` outside the ternary operator
because it is a common filter, no matter if a person is male or female